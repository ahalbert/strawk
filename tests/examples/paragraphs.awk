BEGIN {
  paragraph=0
}
# This regex matches each sentence in the input, and then prints the sentence. It works across newlines!
/(?s)(.*?)\n\n/ {
  paragraph += 1
  if paragraph % 2 == 0 {
    print $0
  }
}
