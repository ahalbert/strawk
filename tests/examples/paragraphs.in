Sunt molestias autem doloremque. Sed ut doloremque occaecati quo est quam numquam 
exercitationem suscipit et. Ad enim voluptatem consequatur vitae quis. Maxime 
quasi magni velit eius nam aut esse voluptatibus quis velit repellendus. 
Temporibus facilis ut porro deleniti excepturi quas alias placeat. Numquam minus 
aut doloribus fugit magni dolorum. Omnis ut minus rem quo est qui voluptate iste 
impedit.

Id eligendi quis ab aliquid impedit tempore velit corrupti. Voluptates fugiat quia 
alias doloribus voluptatem laboriosam possimus quidem repellendus quo aperiam est 
reiciendis culpa. Occaecati ratione est voluptas accusantium eos qui sint consequatur 
maxime. Ad voluptatem sed nihil rem est ipsa aut impedit numquam vel voluptas. Officia 
quod sit ullam dolor placeat aliquid harum modi in sunt qui. Ex est id soluta. Modi 
distinctio ea quia voluptatum corrupti occaecati maxime quia unde voluptatem explicabo 
quam. Porro voluptatem dolor recusandae possimus repudiandae incidunt.

Et nulla rerum deleniti sed labore eos. Distinctio omnis vel illum eos in. Perferendis 
est aliquam saepe dolores fugiat tempore minima molestiae eos adipisci et distinctio 
iste.

Dignissimos provident voluptatem vero eum blanditiis voluptatum. A quaerat voluptas est 
architecto modi. Quasi numquam provident consectetur qui deserunt assumenda sequi impedit 
vitae eaque incidunt et. Qui nobis impedit molestiae omnis ducimus et voluptatem quia.

Occaecati sunt dolore incidunt quas eos suscipit nisi quas similique eaque dolorum. Quos 
consequatur temporibus earum aut dolor aut aut itaque quibusdam quibusdam reiciendis est 
cum. Consequatur at dolorum consequatur nulla at dignissimos ipsam perspiciatis rerum 
nulla enim adipisci veniam ut mollitia. Ducimus non dolorem in fuga quo quo cumque 
corporis aut ex. Rerum pariatur sunt sint quia perspiciatis et sed illo quam modi numquam 
odio aliquid repellendus molestiae.
