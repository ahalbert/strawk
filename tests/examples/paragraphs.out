Id eligendi quis ab aliquid impedit tempore velit corrupti. Voluptates fugiat quia 
alias doloribus voluptatem laboriosam possimus quidem repellendus quo aperiam est 
reiciendis culpa. Occaecati ratione est voluptas accusantium eos qui sint consequatur 
maxime. Ad voluptatem sed nihil rem est ipsa aut impedit numquam vel voluptas. Officia 
quod sit ullam dolor placeat aliquid harum modi in sunt qui. Ex est id soluta. Modi 
distinctio ea quia voluptatum corrupti occaecati maxime quia unde voluptatem explicabo 
quam. Porro voluptatem dolor recusandae possimus repudiandae incidunt.


Dignissimos provident voluptatem vero eum blanditiis voluptatum. A quaerat voluptas est 
architecto modi. Quasi numquam provident consectetur qui deserunt assumenda sequi impedit 
vitae eaque incidunt et. Qui nobis impedit molestiae omnis ducimus et voluptatem quia.


